data/transaction/apiTransactionResult.go

```diff
@@ -0,0 +1,17 @@
+package transaction
+
+// ApiTransactionResult is the DTO which will be returned on the get transaction by hash endpoint
```

Comment:
what means DTO .... dont use shortcuts in comments

Reply:
done